### DISCUSSION TITLE
SMTP Error using localhost

### DISCUSSION THREAD

**Discussion by Koruudo** [nextcloud/all-in-one#3606] [Questions] [answered]
Hello! I recently set up a local SMTP server on the same server that's running Nextcloud AIO, SMTP is running on port 1025. I've confirmed the SMTP server is working correctly w/ authentication through telnet.

However, when I try to connect in Nextcloud, I get the following error:
![image](https://github.com/nextcloud/all-in-one/assets/25572885/5dddc324-b5ec-4d13-8249-c05252944b93)

SMTP Settings:
![image](https://github.com/nextcloud/all-in-one/assets/25572885/f64d7fa7-8c52-456e-a6cb-d049b2d40191)

I've tried the following for the Server address:

- 127.0.0.1
- localhost
- FQDN
- Server Hostname

Any help is greatly appreciated, thank you. :)

**Comment by Zoey2936** [comment, marked as answer]
docker container have their own localhost, so you can't connect from docker to the hosts localhost

**Reply by Koruudo** [reply]
Dang, I had a feeling that's what it was. I think I'd have to do some trickery to get Nextcloud to connect to the cloud somewhere then back to my ISP to the server.

Thanks for confirming!

**Reply by siwilk** [reply]
Did you manage to find a solution to this in the end?

**Reply by Koruudo** [reply]
Unfortunately, I did not. I resorted to using my email provider's SMTP relay for sending email to myself.

You start getting into connecting docker containers to the host and it opens security holes that I'd rather just avoid entirely.
